scalability/nsichneu.c

```diff
@@ -88,11 +88,11 @@ int main()
   P3_is_marked = __llbmc_nondef_int();
 #else
    klee_make_symbolic(P1_marking_member_0, 3 * sizeof(long), "P1_marking_member_0");
-   klee_make_symbolic(&P1_is_marked, sizeof(int), "P1_is_marked");
+/*   klee_make_symbolic(&P1_is_marked, sizeof(int), "P1_is_marked");*/
```

Comment:
@rasoolmaghareh I recommend the commented part for nsichneu.c to be simply deleted to prevent anyone to stop and think about why they are commented out.